mypy/subtypes.py

```diff
@@ -52,6 +53,9 @@ def is_subtype(left: Type, right: Type,
         return any(is_subtype(left, item, type_parameter_checker,
                               ignore_pos_arg_names=ignore_pos_arg_names)
                    for item in right.items)
+    elif isinstance(right, ClassVarType):
+        return is_subtype(left, right.item, type_parameter_checker,
+                          ignore_pos_arg_names=ignore_pos_arg_names)
```

Comment:
I don't think ``ClassVar`` represents actually a new type, so that it should not appear here, otherwise you would need to implement also ``meet_types``, ``join_types``, etc. ``ClassVar`` is rather an "access modifier" for a class member.